src/test-apps/MockTimeSyncClient.cpp

```diff
@@ -338,7 +345,7 @@ void MockTimeSyncClient::SetupConnectionToService(void)
 {
     WEAVE_ERROR err = WEAVE_NO_ERROR;
 
-    CloseConnectionToService();
+    CloseConnectionToService(err);
```

Comment:
Shouldn't this abort the connection?

Reply:
I have reworked this so that it always calls Abort.